src/app/identity.service.ts

```diff
@@ -223,15 +223,19 @@ export class IdentityService {
 
     const {
       id,
-      payload: { encryptedSeedHex, unsignedHashes },
+      payload: { encryptedSeedHex, unsignedHashes, ownerPublicKeyBase58Check },
```

Comment:
Hmm this is a little confusing as we use `ownerPublicKeyBase58Check` to represent the "owner" of a derived key. is there a different term we could use? or am I just confused and it means the same thing?

Reply:
In this case the ownerPublicKey does represent the owner of the derived key, it's just that we need to know which public key bc the previous assumption was the seed hex belongs to account `0`. Now that we have sub accounts we need two pieces of data in order to sign, we need to know the account number (public key is a proxy for that) AND the seed.